### PR TITLE
feat(api): add sms control schema

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/workflows-v2/shared/index.ts
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/shared/index.ts:1]: rename

File: apps/api/src/app/workflows-v2/shared/step-type-to-control.mapper.ts [hunk 1]
```diff
@@ -11,7 +12,7 @@ export const PERMISSIVE_EMPTY_SCHEMA = {
   additionalProperties: true,
 } as JSONSchemaDto;
 
-export const stepTypeToDefaultDashboardControlSchema: Record<ChannelStepEnum | ActionStepEnum, ControlSchemas> = {
+export const stepTypeToControlSchema: Record<ChannelStepEnum | ActionStepEnum, ControlSchemas> = {
```
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/shared/step-type-to-control.mapper.ts:15]: rename

File: apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts:1]: rename

File: apps/api/src/app/workflows-v2/workflow.controller.e2e.ts
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/workflow.controller.e2e.ts:1]: rename

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**